### ISSUE TITLE
allow user to reorder analysis languages

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#341]
Blocked by #340.

The user should be able to reorder analysis languages. It should be possible to do this either through drag and drop, or via keyboard navigation alone.

A good example of reordering items using just keyboard navigation is GitHub projects.
